make.sh

```diff
@@ -347,10 +347,20 @@ test() {
     _fold_end
 
     _fold_start "functional-tests"
+
+    # enter python virtualenv
+    local build_dir="${BUILD_TARGET_DIR}"
```

Comment:
Hmm. Does the python libs have to be host specific (build_target_dir), or host agnostic (build_dir)? (though both point to the same dir, that's the internal context)? 

Also, this currently couples pyenv into build_dir. How about instead, use a var in setup_vars `PYENV=${BUILD_DIR}/.pyenv`

This way it's configurable. And use that var for all py activations without depending on build_dir, so can point to any env without having to change build_dir.

Reply:
Addressed in [161ea46](https://github.com/DeFiCh/ain/pull/2170/commits/161ea46490e7369736f4c93d0c81ce4fb438bfae)